__tests__/api.test.js

```diff
@@ -92,4 +92,55 @@ describe(' GET /api/reviews/:review_id', () => {
             .get('/api/reviews/1000')
             .expect(404)
     });
+});
+
+describe(' 6. GET /api/reviews/:review_id/comments', () => {
+    test('returns correct comments', () => {
+        return request(app)
+            .get('/api/reviews/2/comments')
+            .expect(200)
```

Comment:
Another test is needed for the status code 200 OK - you need a test where the review has no comments so an empty array is returned.